src/Components/Missions.js

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissionsData, reserveMission, leaveMission } from '../redux/missions/missionsSlice';
+
+const Missions = () => {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.missions.missions);
+  const [currentMissionId, setCurrentMissionId] = useState(null);
+
+  useEffect(() => {
+    dispatch(fetchMissionsData());
+  }, [dispatch]);
```

Comment:
- [ ] Kindly make sure the fetched data are not being added to the store whenever a user changes view / uses navigations.

    - ### APP BEHAVIOUR
    
   
![chrome-capture-2023-4-25 (3)](https://github.com/tomdan-ai/Travalers-space-hub/assets/87197412/7a9f92aa-07ea-496d-876b-66a81301e280)


  _**Hint: Make sure to dispatch the `fetchMissionsData()` action only when the `missions`  array in the redux store is empty**_